src/Illuminate/Encryption/EncryptionServiceProvider.php

```diff
@@ -77,9 +77,7 @@ protected function key(array $config)
     {
         return tap($config['key'], function ($key) {
             if (empty($key)) {
-                throw new RuntimeException(
-                    'No application encryption key has been specified.'
-                );
+                throw new MissingKeyException();
```

Comment:
Maybe it would be better to have more concrete name to reflect what kind of key is missing?
`MissingAppKeyException` or `MissingEncryptionKeyException`

Reply:
@antonkomarev  @driesvints which name would you prefer?
`MissingAppKeyException` or `MissingEncryptionKeyException`.
Yesterday, I thought about this too. But at the end I just named it without the specification